<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContactDialog</class>
 <widget class="QDialog" name="AddContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>580</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>110</y>
      <width>650</width>
      <height>360</height>
     </rect>
    </property>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>333</x>
       <y>240</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{background-color:#ffffff;color:#484848;border:1px solid rgb(125,85,40);border-radius:3px;}
QToolButton:hover{color:rgb(125,85,40);}</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="contactLabel2">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>17</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>添加联系人</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>93</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>账户地址：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>153</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>备注名：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="addressLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>400</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="remarkLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>185</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="tipLabel2">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>123</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>240</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{background-color:rgb(125,85,40);color:#ffffff;border:none;border-radius:3px;}
QToolButton:hover{background-color:rgb(185,150,85);}
</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="tipLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>123</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>155</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(177, 177, 177);</string>
     </property>
     <property name="text">
      <string>选填</string>
     </property>
    </widget>
    <widget class="QLabel" name="gifLabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>96</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tipLabel4">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>190</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tipLabel3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>remarkLineEdit</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
